pkg/test/client.go

```diff
@@ -21,6 +21,7 @@ import (
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/util/retry"
 	dynclient "sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
@JAORMX the entry in CHANGELO is no longer here :-( 
Could you please add it again?

Reply:
which entry?

Reply:
An entry describing the change/fix/add to let the users know what was changed. Was not it added before?
See [here](https://github.com/operator-framework/operator-sdk/blob/master/CHANGELOG.md)
Following my suggestion.



```
...
## Changed
- Added retry logic to the cleanup function from the e2e test framework in order to allow it to be achieved in the scenarios where temporary network issues are faced. ([#2277](https://github.com/operator-framework/operator-sdk/pull/2277))
```
WDYT?

Reply:
Sonuds good to me! thanks!